Resources/android/ExampleView.js

```diff
@@ -0,0 +1,20 @@
+var View = require('android.view.View'),
+  Activity = require('android.app.Activity'),
+  TiActivity = new Activity(Ti.Android.currentActivity);
+
+exports.create = function(params) {
+  params = params || {};
+
+  var view = new View(TiActivity);
+
+  if (params.backgroundColor) {
+
+    // Hyperloop should provide cross-platform helpers you can use anywhere in
+    // the app to do stuff like this:
+    var gColor = Hyperloop.colorFromString(params.backgroundColor);
+
+    view.setBackgroundColor(gColor);
+  }
```

Comment:
The object we return from these factories much be _addable_ to Titanium views. Is it safe (in contrast with Titanium proxies) to add custom setters to these?

Do you mean something like this?

```
exports.create = _.defaults(Hyperloop.View, {
  // overrides
});
```